base/index.mustache

```diff
@@ -4,28 +4,32 @@ const fs = require('fs');
 
 require('required_env')(fs.readdirSync('./env').reduce((r, file) => {
   return r.concat(require('./env/'+file));
-}, []))
+}, []));
 
-const restify = require('restify');
+const cors = require('cors');
+const app = require('express')();
+const bodyParser = require('body-parser');
+const server = require('http').createServer(app);
+const io = require('socket.io')(server);
 
-const app = restify.createServer();
-const io = require('socket.io').listen(app.server);
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(cors());
 
-app.use(restify.bodyParser());
-app.use(restify.queryParser());
-
-fs.readdirSync('./middleware').map(name => {
-  require('./middleware/'+name)(app);
-});
```

Comment:
I think it's worth having an opinion about where middleware should go. It saves people _having_ to re-make that decision every time. They're still welcome to make a different decision, because it's just code that they can edit. If we give a sane default, though, they're not forced to.

Also, while we are no longer adding middleware as part of base we're still likely to want redbeard middleware in other generators. User is the main area that springs to mind.